moor_generator/lib/src/writer/queries/query_writer.dart

```diff
@@ -440,7 +440,7 @@ class QueryWriter {
           }
 
           final columnName = column.name.name;
-          expanded.write('"$table.$columnName" AS "$prefix.$columnName"');
+          expanded.write('$table.$columnName AS "$prefix.$columnName"');
```

Comment:
Does it still work if you wrap table and columns in double quotes individually?

```suggestion
          expanded.write('"$table"."$columnName" AS "$prefix.$columnName"');
```

That should be safer in case the table or column name is a reserved keyword.

Reply:
Ah yes. That also works, I think having the quotes there at all and that it worked elsewhere made me forgot how I'd usually quote tables and columns.

Pushed that changed instead.